src/track.ts

```diff
@@ -80,7 +80,15 @@ const getBaseRequestBody = (
         ? Math.floor(client.timestamp / 1e3)
         : client.timestamp,
     event_source_url: payload.dl || client.url.href,
-    data_processing_options: [],
+    ...(settings.dataProcessingOptions && {
+      data_processing_options: ['LDU'],
```

Comment:
This will change the current behavior for all users, changing the default from `data_processing_options: []` to `data_processing_options` being undefined. I'm not sure if that's intended? I have some vague memory that this empty array was needed.

Reply:
Also, wouldn't letting the user specify `LDU` explicitly be more future-proof, in case FB adds some additional options?

Reply:
> This will change the current behavior for all users, changing the default from `data_processing_options: []` to `data_processing_options` being undefined. I'm not sure if that's intended? I have some vague memory that this empty array was needed.

Facebook say you can either send as an empty array, or remove the parameter, [quote](https://developers.facebook.com/docs/marketing-apis/data-processing-options/): "to explicitly not enable Limited Data Use (LDU), specify an empty array for each event or simply remove the field in the payload". 

> Also, wouldn't letting the user specify LDU explicitly be more future-proof, in case FB adds some additional options?

Yes, you are right. I will change it.

Reply:
Got it. I saw "An empty array can be sent to explicitly specify that this event or record shouldn't be processed with the Limited Data Use restrictions." and thought that we would be changing the default behavior, but I guess it's the same then